### ISSUE TITLE
OpenSLO

### ISSUE THREAD

**Issue by mhausenblas** [slok/sloth#74]
Great tool, thanks! Can you share your thoughts around supporting [OpenSLO](https://github.com/OpenSLO/OpenSLO) please?

**Comment by slok** [comment]
Hi @mhausenblas!

I was asked that question last week on Twitter, so I think that is something that people are interested in :)

tl;dr: I would love to!

Sloth at this moment is focused on a very easy spec (e.g: no custom time windows). However, OpenSLO has a lot of knobs (that not every people want or know how to set). So... I need to research a bit and see if I can map OpenSLO to the current Sloth generator implementation. If this can be done, what I was thinking, is supporting OpenSLO spec and support only the options types that Sloth supports (e.f 30d time windows, Prometheus...).

With time we would have more information and we could see what new features could Sloth support from the OpenSLO spec.

How does this sound?

**Comment by kitmerker** [comment]
would be cool to have you in the openslo slack, maybe we can brainstorm there how to make this work?

**Comment by slok** [comment]
> would be cool to have you in the openslo slack, maybe we can brainstorm there how to make this work?

That would be awesome! thanks.

**Comment by mhausenblas** [comment]
Thanks so much @slok and that sounds like a great plan to me! As Kit suggested, maybe having a bit of an informal convo on Slack, to scope and set expectations right won't hurt ;)

**Comment by slok** [comment]
@mhausenblas @kitmerker I've added initial support for OpenSLO :tada:, Any feedback is much appreciated!
